src/devices/controller.c

```diff
@@ -19,6 +19,7 @@
 #include "vdml/vdml.h"
 
 #define CONTROLLER_MAX_COLS 15
+#define CONTROLLER_MAX_CHARS 31
```

Comment:
A couple of nits: 
- Might as well get rid of CONTROLLER_MAX_COLS
- Parenthesize and unsign the define

Other than that this looks good, thanks for the contribution :) 

```suggestion
#define CONTROLLER_MAX_CHARS ( 31U )
```

Reply:
Addressed in https://github.com/purduesigbots/pros/pull/683/commits/f907d62932d81042b081eac8641ee435cb3e5e18. (`CONTROLLER_MAX_COLS` still needs to be there since the controller only supports 20 columns, but can print 31 characters, due to the font being proportional).